packages/sanes-chrome-extension/config/development.json

```diff
@@ -1,10 +1,9 @@
 {
-  "chains":
-  [
+  "chains": [
     {
       "chainSpec": {
         "codecType": "bns",
-        "bootstrapNodes": ["wss://bns.hugnet.iov.one/"]
```

Comment:
This change makes unit tests to fail because jsdom raises CORS errors. Options:
- ~Do not use react-scripts and call JEST directly after tweaking CORS stuff~
- ~Tweak and dig --env=node in unit testing. Need to fix react-scripts errors because it expects jsdom~
- Create another file called `config/unit.json` using websockets and update package.json' unit-test script.

My preferred option is the third one, otherwise, the effort done for keeping minimal the number of libraries needed would be for nothing.

Reply:
The CORS issue is basically a misconfuguration of the local BNS/BCP blockchains. It was never detected before because we always connected via WebSockets to those chains. The last commit fixes the blockchain configuration to allow all clients.

Some other test fails are probably left for next week but the CORS issue should be solved.